app/Services/AssignUserToOrganizationsService.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Http\Integrations\GithubConnector;
+use App\Http\Integrations\Requests\GetUsersOrganizationsRequest;
+use App\Models\Organization;
+use App\Models\User;
+
+class AssignUserToOrganizationsService
+{
+    public function __construct(
+        protected GithubConnector $githubConnector,
+    ) {}
+
+    public function assign(User $user): void
+    {
+        $request = new GetUsersOrganizationsRequest($user);
+
+        $response = $this->githubConnector->send($request);
```

Comment:
U can use https://docs.saloon.dev/digging-deeper/data-transfer-objects for handling data from response